puppet/modules/kubernetes/manifests/kubelet.pp

```diff
@@ -180,6 +183,27 @@
     $_ca_file = $ca_file
   }
 
+  exec {'sysctl-system':
+    command     => 'sysctl --system',
+    refreshonly => true,
+    path        => ['/usr/bin/', '/bin', '/usr/sbin'],
```

Comment:
If you use `$::kubernetes::path` here it should also work on ubuntu as they have the `sysctl` bin in `/sbin`

Reply:
Thanks, missed this yesterday